pkg/boot/gboot/entry/entry.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2020 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package entry implements config file parsing for GRUB entry files. An entry
```

Comment:
this actually would have been a good candidate for pkg/grub. env files are a general grub concept.